src/io/flutter/run/bazelTest/FlutterBazelTestConfigurationEditorForm.java

```diff
@@ -56,4 +95,48 @@ protected void applyEditorTo(@NotNull final BazelTestConfig configuration) {
   protected JComponent createEditor() {
     return myMainPanel;
   }
+
+
+  /**
+   * When switching between file and directory scope, update the next field to
+   * a suitable default.
+   */
+  private void updateFields(Scope next) {
+    if (next == Scope.TARGET_PATTERN && displayedScope != Scope.TARGET_PATTERN) {
+    }
+    else if (next != Scope.TARGET_PATTERN) {
+      if (myEntryFile.getText().isEmpty()) {
+        myEntryFile.setText(myBuildTarget.getText().replace("//", ""));
```

Comment:
Do you need a call to StringUtil.nullize() like you have in line 84?

Reply:
Not necessary, but it's a good idea.

This if statement should semantically mean that the text in myEntryFile will be the same as passing a null value.  To achieve this, it's best to use the same logic for determining its value as we were using in the other line.